Events not processed

### Self-Hosted Version

25.1.0

### CPU Architecture

x86_64

### Docker Version

26.1.3

### Docker Compose Version

2.27.0

### Machine Specification

- [x] My system meets the minimum system requirements of Sentry

### Steps to Reproduce

1. Submit a test event using a simple script:
```
import sentry_sdk
sentry_sdk.init(
    "https://my-dsn-here@sentry.localdomain.org/1",
    traces_sample_rate=1.0,
    debug=True
)
print(1/0)
```

### Expected Result

The error appears in the UI.

### Actual Result

Nothing appears in the UI. The only error in the logs appears to be unrelated because it references a connection attempt to `https://openaipublic.blob.core.windows.net/encodings/cl100k_base.tiktoken` which fails because we are running Sentry in isolated environment:
```
worker-1                                        | 08:43:31 [WARNING] celery.worker.request: Soft time limit (60s) exceeded for sentry.tasks.store.save_event[826034e5-6289-46d1-8377-798cd2aab653]
worker-1                                        | Traceback (most recent call last):
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/celery/app/trace.py", line 477, in trace_task
worker-1                                        |     R = retval = fun(*args, **kwargs)
worker-1                                        |                  ~~~^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/utils.py", line 1860, in runner
worker-1                                        |     return sentry_patched_function(*args, **kwargs)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/integrations/celery/__init__.py", line 416, in _inner
worker-1                                        |     reraise(*exc_info)
worker-1                                        |     ~~~~~~~^^^^^^^^^^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/utils.py", line 1795, in reraise
worker-1                                        |     raise value
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/integrations/celery/__init__.py", line 411, in _inner
worker-1                                        |     return f(*args, **kwargs)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/celery/app/trace.py", line 760, in __protected_call__
worker-1                                        |     return self.run(*args, **kwargs)
worker-1                                        |            ~~~~~~~~^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/silo/base.py", line 158, in override
worker-1                                        |     return original_method(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/tasks/base.py", line 118, in _wrapped
worker-1                                        |     result = func(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/tasks/store.py", line 634, in save_event
worker-1                                        |     _do_save_event(
worker-1                                        |     ~~~~~~~~~~~~~~^
worker-1                                        |         cache_key,
worker-1                                        |         ^^^^^^^^^^
worker-1                                        |     ...<5 lines>...
worker-1                                        |         **kwargs,
worker-1                                        |         ^^^^^^^^^
worker-1                                        |     )
worker-1                                        |     ^
worker-1                                        |   File "/usr/src/sentry/src/sentry/tasks/store.py", line 564, in _do_save_event
worker-1                                        |     manager.save(
worker-1                                        |     ~~~~~~~~~~~~^
worker-1                                        |         project_id,
worker-1                                        |         ^^^^^^^^^^^
worker-1                                        |     ...<3 lines>...
worker-1                                        |         has_attachments=has_attachments,
worker-1                                        |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |     )
worker-1                                        |     ^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/tracing_utils.py", line 713, in func_with_tracing
worker-1                                        |     return func(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/event_manager.py", line 487, in save
worker-1                                        |     return self.save_error_events(
worker-1                                        |            ~~~~~~~~~~~~~~~~~~~~~~^
worker-1                                        |         project,
worker-1                                        |         ^^^^^^^^
worker-1                                        |     ...<5 lines>...
worker-1                                        |         has_attachments=has_attachments,
worker-1                                        |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |     )
worker-1                                        |     ^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/tracing_utils.py", line 713, in func_with_tracing
worker-1                                        |     return func(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/event_manager.py", line 535, in save_error_events
worker-1                                        |     group_info = assign_event_to_group(event=job["event"], job=job, metric_tags=metric_tags)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/sentry_sdk/tracing_utils.py", line 713, in func_with_tracing
worker-1                                        |     return func(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/event_manager.py", line 1270, in assign_event_to_group
worker-1                                        |     primary = get_hashes_and_grouphashes(job, run_primary_grouping, metric_tags)
worker-1                                        |   File "/usr/src/sentry/src/sentry/event_manager.py", line 1347, in get_hashes_and_grouphashes
worker-1                                        |     grouping_config, hashes, variants = hash_calculation_function(project, job, metric_tags)
worker-1                                        |                                         ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/ingest/hashing.py", line 172, in run_primary_grouping
worker-1                                        |     hashes, variants = _calculate_primary_hashes_and_variants(project, job, grouping_config)
worker-1                                        |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/ingest/hashing.py", line 185, in _calculate_primary_hashes_and_variants
worker-1                                        |     return _calculate_event_grouping(project, job["event"], grouping_config)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/ingest/hashing.py", line 74, in _calculate_event_grouping
worker-1                                        |     hashes, variants = event.get_hashes_and_variants(loaded_grouping_config)
worker-1                                        |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/eventstore/models.py", line 338, in get_hashes_and_variants
worker-1                                        |     variants = self.get_grouping_variants(config)
worker-1                                        |   File "/usr/src/sentry/src/sentry/eventstore/models.py", line 447, in get_grouping_variants
worker-1                                        |     return get_grouping_variants_for_event(self, loaded_grouping_config)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/api.py", line 398, in get_grouping_variants_for_event
worker-1                                        |     strategy_component_variants: dict[str, ComponentVariant] = _get_variants_from_strategies(
worker-1                                        |                                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
worker-1                                        |         event, context
worker-1                                        |         ^^^^^^^^^^^^^^
worker-1                                        |     )
worker-1                                        |     ^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/api.py", line 295, in _get_variants_from_strategies
worker-1                                        |     current_strategy_components_by_variant = strategy.get_grouping_components(
worker-1                                        |         event, context=context
worker-1                                        |     )
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 242, in get_grouping_components
worker-1                                        |     components_by_variant = self.get_grouping_component(event, variant=None, context=context)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 234, in get_grouping_component
worker-1                                        |     return self(event=event, context=context, *args)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 212, in __call__
worker-1                                        |     return self._invoke(self.func, *args, **kwargs)
worker-1                                        |            ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 209, in _invoke
worker-1                                        |     return func(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/newstyle.py", line 614, in chained_exception
worker-1                                        |     id(exception): context.get_grouping_components_by_variant(exception, event=event, **meta)
worker-1                                        |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 128, in get_grouping_components_by_variant
worker-1                                        |     return self._get_strategy_dict(interface, event=event, **kwargs)
worker-1                                        |            ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 166, in _get_strategy_dict
worker-1                                        |     rv = strategy(interface, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 212, in __call__
worker-1                                        |     return self._invoke(self.func, *args, **kwargs)
worker-1                                        |            ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 209, in _invoke
worker-1                                        |     return func(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/newstyle.py", line 573, in single_exception
worker-1                                        |     normalized = normalize_message_for_grouping(
worker-1                                        |         raw, event, share_analytics=(not favors_other_component)
worker-1                                        |     )
worker-1                                        |   File "/usr/src/sentry/src/sentry/utils/metrics.py", line 229, in inner
worker-1                                        |     return f(*args, **kwargs)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/strategies/message.py", line 82, in normalize_message_for_grouping
worker-1                                        |     normalized = parameterizer.parameterize_all(trimmed, _shoudl_run_experiment)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 368, in parameterize_all
worker-1                                        |     return self.parametrize_w_experiments(self.parametrize_w_regex(content), should_run)
worker-1                                        |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 356, in parametrize_w_experiments
worker-1                                        |     content = experiment.run(content, _incr_counter)
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 200, in run
worker-1                                        |     content, count = self.apply(content)
worker-1                                        |                      ~~~~~~~~~~^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 264, in replace_uniq_ids_in_str
worker-1                                        |     if _UniqueId.is_probably_uniq_id(s):
worker-1                                        |        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 248, in is_probably_uniq_id
worker-1                                        |     token_length_ratio = _UniqueId.num_tokens_from_string(token_str) / len(token_str)
worker-1                                        |                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 228, in num_tokens_from_string
worker-1                                        |     num_tokens = len(_UniqueId.tiktoken_encoding().encode(token_str))
worker-1                                        |                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
worker-1                                        |   File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 223, in tiktoken_encoding
worker-1                                        |     return tiktoken.get_encoding("cl100k_base")
worker-1                                        |            ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/tiktoken/registry.py", line 86, in get_encoding
worker-1                                        |     enc = Encoding(**constructor())
worker-1                                        |                      ~~~~~~~~~~~^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/tiktoken_ext/openai_public.py", line 76, in cl100k_base
worker-1                                        |     mergeable_ranks = load_tiktoken_bpe(
worker-1                                        |         "https://openaipublic.blob.core.windows.net/encodings/cl100k_base.tiktoken",
worker-1                                        |         expected_hash="223921b76ee99bde995b7ff738513eef100fb51d18c93597a113bcffe865b2a7",
worker-1                                        |     )
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/tiktoken/load.py", line 144, in load_tiktoken_bpe
worker-1                                        |     contents = read_file_cached(tiktoken_bpe_file, expected_hash)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/tiktoken/load.py", line 63, in read_file_cached
worker-1                                        |     contents = read_file(blobpath)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/tiktoken/load.py", line 24, in read_file
worker-1                                        |     resp = requests.get(blobpath)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/requests/api.py", line 73, in get
worker-1                                        |     return request("get", url, params=params, **kwargs)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/requests/api.py", line 59, in request
worker-1                                        |     return session.request(method=method, url=url, **kwargs)
worker-1                                        |            ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/requests/sessions.py", line 589, in request
worker-1                                        |     resp = self.send(prep, **send_kwargs)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/requests/sessions.py", line 703, in send
worker-1                                        |     r = adapter.send(request, **kwargs)
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/requests/adapters.py", line 667, in send
worker-1                                        |     resp = conn.urlopen(
worker-1                                        |         method=request.method,
worker-1                                        |     ...<9 lines>...
worker-1                                        |         chunked=chunked,
worker-1                                        |     )
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/urllib3/connectionpool.py", line 789, in urlopen
worker-1                                        |     response = self._make_request(
worker-1                                        |         conn,
worker-1                                        |     ...<10 lines>...
worker-1                                        |         **response_kw,
worker-1                                        |     )
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/urllib3/connectionpool.py", line 466, in _make_request
worker-1                                        |     self._validate_conn(conn)
worker-1                                        |     ~~~~~~~~~~~~~~~~~~~^^^^^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/urllib3/connectionpool.py", line 1095, in _validate_conn
worker-1                                        |     conn.connect()
worker-1                                        |     ~~~~~~~~~~~~^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/urllib3/connection.py", line 615, in connect
worker-1                                        |     self.sock = sock = self._new_conn()
worker-1                                        |                        ~~~~~~~~~~~~~~^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/urllib3/connection.py", line 196, in _new_conn
worker-1                                        |     sock = connection.create_connection(
worker-1                                        |         (self._dns_host, self.port),
worker-1                                        |     ...<2 lines>...
worker-1                                        |         socket_options=self.socket_options,
worker-1                                        |     )
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/urllib3/util/connection.py", line 73, in create_connection
worker-1                                        |     sock.connect(sa)
worker-1                                        |     ~~~~~~~~~~~~^^^^
worker-1                                        |   File "/.venv/lib/python3.13/site-packages/billiard/pool.py", line 228, in soft_timeout_sighandler
worker-1                                        |     raise SoftTimeLimitExceeded()
worker-1                                        | billiard.exceptions.SoftTimeLimitExceeded: SoftTimeLimitExceeded()
worker-1                                        | 08:43:31 [ERROR] celery.app.trace: Task sentry.tasks.store.save_event[826034e5-6289-46d1-8377-798cd2aab653] raised unexpected: SoftTimeLimitExceeded() (data={'hostname': 'celery@486918f97652', 'id': '826034e5-6289-46d1-8377-798cd2aab653', 'name': 'sentry.tasks.store.save_event', 'exc': 'SoftTimeLimitExceeded()', 'traceback': 'Traceback (most recent call last):\n  File "/.venv/lib/python3.13/site-packages/celery/app/trace.py", line 477, in trace_task\n    R = retval = fun(*args, **kwargs)\n                 ~~~^^^^^^^^^^^^^^^^^\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/utils.py", line 1860, in runner\n    return sentry_patched_function(*args, **kwargs)\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/integrations/celery/__init__.py", line 416, in _inner\n    reraise(*exc_info)\n    ~~~~~~~^^^^^^^^^^^\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/utils.py", line 1795, in reraise\n    raise value\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/integrations/celery/__init__.py", line 411, in _inner\n    return f(*args, **kwargs)\n  File "/.venv/lib/python3.13/site-packages/celery/app/trace.py", line 760, in __protected_call__\n    return self.run(*args, **kwargs)\n           ~~~~~~~~^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/silo/base.py", line 158, in override\n    return original_method(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/tasks/base.py", line 118, in _wrapped\n    result = func(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/tasks/store.py", line 634, in save_event\n    _do_save_event(\n    ~~~~~~~~~~~~~~^\n        cache_key,\n        ^^^^^^^^^^\n    ...<5 lines>...\n        **kwargs,\n        ^^^^^^^^^\n    )\n    ^\n  File "/usr/src/sentry/src/sentry/tasks/store.py", line 564, in _do_save_event\n    manager.save(\n    ~~~~~~~~~~~~^\n        project_id,\n        ^^^^^^^^^^^\n    ...<3 lines>...\n        has_attachments=has_attachments,\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/tracing_utils.py", line 713, in func_with_tracing\n    return func(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/event_manager.py", line 487, in save\n    return self.save_error_events(\n           ~~~~~~~~~~~~~~~~~~~~~~^\n        project,\n        ^^^^^^^^\n    ...<5 lines>...\n        has_attachments=has_attachments,\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/tracing_utils.py", line 713, in func_with_tracing\n    return func(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/event_manager.py", line 535, in save_error_events\n    group_info = assign_event_to_group(event=job["event"], job=job, metric_tags=metric_tags)\n  File "/.venv/lib/python3.13/site-packages/sentry_sdk/tracing_utils.py", line 713, in func_with_tracing\n    return func(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/event_manager.py", line 1270, in assign_event_to_group\n    primary = get_hashes_and_grouphashes(job, run_primary_grouping, metric_tags)\n  File "/usr/src/sentry/src/sentry/event_manager.py", line 1347, in get_hashes_and_grouphashes\n    grouping_config, hashes, variants = hash_calculation_function(project, job, metric_tags)\n                                        ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/ingest/hashing.py", line 172, in run_primary_grouping\n    hashes, variants = _calculate_primary_hashes_and_variants(project, job, grouping_config)\n                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/ingest/hashing.py", line 185, in _calculate_primary_hashes_and_variants\n    return _calculate_event_grouping(project, job["event"], grouping_config)\n  File "/usr/src/sentry/src/sentry/grouping/ingest/hashing.py", line 74, in _calculate_event_grouping\n    hashes, variants = event.get_hashes_and_variants(loaded_grouping_config)\n                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/eventstore/models.py", line 338, in get_hashes_and_variants\n    variants = self.get_grouping_variants(config)\n  File "/usr/src/sentry/src/sentry/eventstore/models.py", line 447, in get_grouping_variants\n    return get_grouping_variants_for_event(self, loaded_grouping_config)\n  File "/usr/src/sentry/src/sentry/grouping/api.py", line 398, in get_grouping_variants_for_event\n    strategy_component_variants: dict[str, ComponentVariant] = _get_variants_from_strategies(\n                                                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        event, context\n        ^^^^^^^^^^^^^^\n    )\n    ^\n  File "/usr/src/sentry/src/sentry/grouping/api.py", line 295, in _get_variants_from_strategies\n    current_strategy_components_by_variant = strategy.get_grouping_components(\n        event, context=context\n    )\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 242, in get_grouping_components\n    components_by_variant = self.get_grouping_component(event, variant=None, context=context)\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 234, in get_grouping_component\n    return self(event=event, context=context, *args)\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 212, in __call__\n    return self._invoke(self.func, *args, **kwargs)\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 209, in _invoke\n    return func(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/grouping/strategies/newstyle.py", line 614, in chained_exception\n    id(exception): context.get_grouping_components_by_variant(exception, event=event, **meta)\n                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 128, in get_grouping_components_by_variant\n    return self._get_strategy_dict(interface, event=event, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 166, in _get_strategy_dict\n    rv = strategy(interface, **kwargs)\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 212, in __call__\n    return self._invoke(self.func, *args, **kwargs)\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/strategies/base.py", line 209, in _invoke\n    return func(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/grouping/strategies/newstyle.py", line 573, in single_exception\n    normalized = normalize_message_for_grouping(\n        raw, event, share_analytics=(not favors_other_component)\n    )\n  File "/usr/src/sentry/src/sentry/utils/metrics.py", line 229, in inner\n    return f(*args, **kwargs)\n  File "/usr/src/sentry/src/sentry/grouping/strategies/message.py", line 82, in normalize_message_for_grouping\n    normalized = parameterizer.parameterize_all(trimmed, _shoudl_run_experiment)\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 368, in parameterize_all\n    return self.parametrize_w_experiments(self.parametrize_w_regex(content), should_run)\n           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 356, in parametrize_w_experiments\n    content = experiment.run(content, _incr_counter)\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 200, in run\n    content, count = self.apply(content)\n                     ~~~~~~~~~~^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 264, in replace_uniq_ids_in_str\n    if _UniqueId.is_probably_uniq_id(s):\n       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 248, in is_probably_uniq_id\n    token_length_ratio = _UniqueId.num_tokens_from_string(token_str) / len(token_str)\n                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 228, in num_tokens_from_string\n    num_tokens = len(_UniqueId.tiktoken_encoding().encode(token_str))\n                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File "/usr/src/sentry/src/sentry/grouping/parameterization.py", line 223, in tiktoken_encoding\n    return tiktoken.get_encoding("cl100k_base")\n           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File "/.venv/lib/python3.13/site-packages/tiktoken/registry.py", line 86, in get_encoding\n    enc = Encoding(**constructor())\n                     ~~~~~~~~~~~^^\n  File "/.venv/lib/python3.13/site-packages/tiktoken_ext/openai_public.py", line 76, in cl100k_base\n    mergeable_ranks = load_tiktoken_bpe(\n        "https://openaipublic.blob.core.windows.net/encodings/cl100k_base.tiktoken",\n        expected_hash="223921b76ee99bde995b7ff738513eef100fb51d18c93597a113bcffe865b2a7",\n    )\n  File "/.venv/lib/python3.13/site-packages/tiktoken/load.py", line 144, in load_tiktoken_bpe\n    contents = read_file_cached(tiktoken_bpe_file, expected_hash)\n  File "/.venv/lib/python3.13/site-packages/tiktoken/load.py", line 63, in read_file_cached\n    contents = read_file(blobpath)\n  File "/.venv/lib/python3.13/site-packages/tiktoken/load.py", line 24, in read_file\n    resp = requests.get(blobpath)\n  File "/.venv/lib/python3.13/site-packages/requests/api.py", line 73, in get\n    return request("get", url, params=params, **kwargs)\n  File "/.venv/lib/python3.13/site-packages/requests/api.py", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/.venv/lib/python3.13/site-packages/requests/sessions.py", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n  File "/.venv/lib/python3.13/site-packages/requests/sessions.py", line 703, in send\n    r = adapter.send(request, **kwargs)\n  File "/.venv/lib/python3.13/site-packages/requests/adapters.py", line 667, in send\n    resp = conn.urlopen(\n        method=request.method,\n    ...<9 lines>...\n        chunked=chunked,\n    )\n  File "/.venv/lib/python3.13/site-packages/urllib3/connectionpool.py", line 789, in urlopen\n    response = self._make_request(\n        conn,\n    ...<10 lines>...\n        **response_kw,\n    )\n  File "/.venv/lib/python3.13/site-packages/urllib3/connectionpool.py", line 466, in _make_request\n    self._validate_conn(conn)\n    ~~~~~~~~~~~~~~~~~~~^^^^^^\n  File "/.venv/lib/python3.13/site-packages/urllib3/connectionpool.py", line 1095, in _validate_conn\n    conn.connect()\n    ~~~~~~~~~~~~^^\n  File "/.venv/lib/python3.13/site-packages/urllib3/connection.py", line 615, in connect\n    self.sock = sock = self._new_conn()\n                       ~~~~~~~~~~~~~~^^\n  File "/.venv/lib/python3.13/site-packages/urllib3/connection.py", line 196, in _new_conn\n    sock = connection.create_connection(\n        (self._dns_host, self.port),\n    ...<2 lines>...\n        socket_options=self.socket_options,\n    )\n  File "/.venv/lib/python3.13/site-packages/urllib3/util/connection.py", line 73, in create_connection\n    sock.connect(sa)\n    ~~~~~~~~~~~~^^^^\n  File "/.venv/lib/python3.13/site-packages/billiard/pool.py", line 228, in soft_timeout_sighandler\n    raise SoftTimeLimitExceeded()\nbilliard.exceptions.SoftTimeLimitExceeded: SoftTimeLimitExceeded()\n', 'args': '()', 'kwargs': "{'cache_key': 'e:cfbf1091919c4e65814129027b8e05e4:1', 'data': None, 'start_time': 1743756151.0, 'event_id': 'cfbf1091919c4e65814129027b8e05e4', 'project_id': 1, '__start_time': 1743756151.91561}", 'description': 'raised unexpected', 'internal': False})
```
But this error appears every time after I am submitting a test event. Are we doing something wrong? Please note that version 23.12.1 worked fine for us, and all we did was the upgrade to 25.1.0 (following the hard stops).

### Event ID

_No response_

--- a/src/sentry/grouping/strategies/message.py
+++ b/src/sentry/grouping/strategies/message.py
@@ -15,6 +15,7 @@ from sentry.grouping.strategies.base import (
 from sentry.interfaces.message import Message
 from sentry.options.rollout import in_rollout_group
 from sentry.utils import metrics
+from sentry.utils.settings import is_self_hosted
 
 if TYPE_CHECKING:
     from sentry.eventstore.models import Event
@@ -57,7 +58,8 @@ def normalize_message_for_grouping(message: str, event: Event, share_analytics: 
 
     def _shoudl_run_experiment(experiment_name: str) -> bool:
         return bool(
-            event.project_id
+            not is_self_hosted()
+            and event.project_id
             and (
                 in_rollout_group(
                     f"grouping.experiments.parameterization.{experiment_name}", event.project_id

